.evergreen.yaml

```diff
@@ -20,6 +20,22 @@ functions:
         working_dir: src/atlas-service-broker
         script: |
           go vet ./...
+  "setup_snyk":
+    - command: shell.exec
+      type: system
+      params:
+        script: |
+          curl -L -o snyk https://github.com/snyk/snyk/releases/download/v1.216.0/snyk-linux
+          chmod +x snyk
+          mv snyk /usr/local/bin/
```

Comment:
please don't do this -- all evergreen file access should be within the workspace dir that the tasks run in. The machines run many tasks before being shut down, so changes to the root filesystem will affect future runs.

Instead: 
`````mkdir bin
export PATH="$(pwd)/bin:$PATH"
```

Reply:
That's a very good point, thanks! Fixed now.